PowerEditor/src/Notepad_plus.cpp

```diff
@@ -1811,6 +1833,37 @@ bool Notepad_plus::findInCurrentFile()
 	return true;
 }
 
+void Notepad_plus::showIgnoredFilesMessage(ProcessOperation op, const std::vector<generic_string>& ignoredFiles)
+{
+	generic_string message;
+	switch (op)
+	{
+	case ProcessFindAll:
+		message += ignoredFiles.size() > 1 ?

```

Comment:
Does n++ account for grammar of non-English languages? In pretty much all Slavic languages this would require more than two strings (_1 plik, 2-4 pliki, 5+ plików_) but unless n++ actually accounts for that elsewhere, no point in worrying about this.